src/wp-includes/formatting.php

```diff
@@ -2779,7 +2779,10 @@ function trailingslashit( $string ) {
  * @return string String without the trailing slashes.
  */
 function untrailingslashit( $string ) {
-	return rtrim( $string, '/\\' );
+	if ( ! empty( $string ) ) {
+		$string = rtrim( $string, '/\\' );
+	}
+	return $string;
```

Comment:
This appears to be a change related to another ticket. It should be removed from this PR.

Instead, you should create a new branch from `trunk` for each ticket that you're working on. :slightly_smiling_face:

I see that you opened this PR from changes to your fork's `trunk` branch. Instead, I would recommend changing this branch now and resetting `trunk` to its original state to prevent problems for yourself going forward.

Something like this:

1. Create a new branch from `trunk` called `installation_verify_nonce`:

```sh
git checkout trunk
git checkout -b installation_verify_nonce
```
2. Hard reset `trunk` back to the original state:

```sh
git checkout trunk
git reset --hard HEAD~8
```
3. Clean up the `installation_verify_nonce` branch:

```sh
git checkout installation_verify_nonce
git reset --hard HEAD~3
```
4. Push `installation_verify_nonce`:

```sh
git checkout installation_verify_nonce
git push -u origin installation_verify_nonce
```

5. Now, here on GitHub, edit this PR (beside the title) and set your branch to `installation_verify_nonce`.

6. Push the reset `trunk` to your fork:
```sh
git checkout trunk
git push --force
```

After that, you should be okay to continue contributing by keeping your fork synced with upstream (if using GH CLI: `gh repo sync hiren1094/wordpress-develop`) and opening a new branch for each ticket you want to patch.

Feel free to ping me on Slack (same username) if you run into any issues. :slightly_smiling_face: